lib/ReactViewModels/ViewState.js

```diff
@@ -122,6 +127,28 @@ export default class ViewState {
       }
     });
 
+    // Reflect preview id from terria when loaded from an init source
+
+    this._sharedFromExplorerPanelSubscription = knockout
+      .getObservable(terria, "sharedFromExplorerPanel")
+      .subscribe(sharedFromExplorerPanel => {
+        if (defined(sharedFromExplorerPanel)) {
```

Comment:
We don't care if it's true? Only that it's defined?

Reply:
Thanks good pickup, I think my idea was the property only ever gets set if it is shared, but I've also now updated to check it's also true